sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -1098,7 +1098,7 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
           Literal(Date.valueOf("2018-01-05")),
           Literal(Period.ofDays(2))),
         EmptyRow,
-        "sequence step must be a day year-month interval if start and end values are dates")
+        "sequence step must be a day interval year to month if start and end values are dates")
```

Comment:
@beliefer The error message confuses me slightly, especially the combination `a day interval year to month`. Could you open a PR to improve the error, please, something like "... sequence step must be an interval of day granularity ...".

Reply:
I see.